### ISSUE TITLE
Ignoring Specific Components

### ISSUE THREAD

**Issue by PaulHMason** [PolymerElements/iron-component-page#79]
I've got a composite Polymer control that is composed of a number of elements, some of which are only used internally. Is there any way of preventing a particular control (html import) from appearing in the documentation. I have a wrapper html import that just imports the components that I want to expose documentation for, but the internal ones also appear in the documentation because they're sub imports.

**Comment by ebidel** [comment]
@garlicnation ?

**Comment by blasten** [comment]
I don't think so. It sounds like you need to create two elements e.g (x-internal and x-public) and share some common functionality among the two. In that way, you can only publish the docs for x-public.
